pandas/tests/arrays/string_/test_string.py

```diff
@@ -269,3 +267,31 @@ def test_value_counts_na():
     result = arr.value_counts(dropna=True)
     expected = pd.Series([2, 1], index=["a", "b"], dtype="Int64")
     tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize("func", ["min", "max"])
+@pytest.mark.parametrize("skipna", [True, False])
+def test_reduction(func):
+    strings = ["x", "y", "z"]
+    arr = pd.Series(strings, dtype="string")
+    ser = pd.Series(strings)
+
+    result = getattr(arr, func)(skipna=skipna)
+    expected = getattr(ser, func)(skipna=skipna)
+
+    assert result == expected
+
+
+@pytest.mark.parametrize("func", ["min", "max"])
+@pytest.mark.parametrize("skipna", [True, False])
+def test_reduction_with_na(func, skipna):
```

Comment:
any chance of this being parametrized over StringArray/Index/Series?  if so, these might belong in tests.reductions

Reply:
As written right now I think this would only work with Series. Perhaps these methods should get implemented correctly for StringArray, and then delegate to those within _reduce instead of using the parent class methods (then test both StringArray and Series at once)?

Reply:
Updated to parametrize over StringArray / Series